internal/store/runtime_state.go

```diff
@@ -111,7 +111,7 @@ func (s K8sRuntimeState) RuntimeStatusError() error {
 }
 
 func (s K8sRuntimeState) RuntimeStatus() model.RuntimeStatus {
-	if s.IsUnresourced {
+	if s.IsUnresourced || len(s.Pods) == 0 {
```

Comment:
hmmm....this isn't right.

This means "we haven't seen any pods yet"

what you want is "we will never see any pods"

the distinction is important, because a lot of things will break if you report ready status prematurely (tilt ci, resource deps, etc)

Reply:
@nicks gotcha, is there a built-in way to do that as of today?

Reply:
that's what Unresourced is....i think you'll have to replace Unresourced with a new signifier for this?

Reply:
i think you still need to remove this pod check (here and above)?